vendor/assets/stylesheets/rolodex/settings/utilities/lib/_margin.sass

```diff
@@ -0,0 +1,109 @@
+/*
+  Margin
+  ========================================
+
+  shorthand:
+     .m-        (margin)
+    .mt-        (margin top)
+    .mr-        (margin right)
+    .mb-        (margin bottom)
+    .ml-        (margin left)
+    .mx-        (x-axis (left and right))
+    .my-        (y-axis (top and bottom))
+
+  modifiers:
+    -baseline   ($baselines,   ex: .mt-large, .mb-x-small)
+    -h          ($gutter-half, ex: .mx-h)
+    -0          (reset,        ex: .mt-0)
+
+  resets:
+     .m-0
+    .mt-0
+    .mr-0
+    .mb-0
+    .ml-0
+*/
+
+// Margin from $baseline variables
```

Comment:
we could instead do this like:

```scss
.mt0 - 0
.mt1 - $baseline-steo
.mt2 - $baseline-x-small
.mt3 - $baseline-small
.mt4 - $baseline-base
.mt5 - $baseline-large
.mt6 - $baseline-x-large
.mt7 - $baseline-xx-large
```

I think I prefer the `-baseline` way however